lib/ISAACConfig.cmake

```diff
@@ -64,7 +64,7 @@ if (ISAAC_JPEG)
         endif()
 endif (ISAAC_JPEG)
 
-set(ISAAC_VECTOR_ELEM "2" CACHE STRING "The amounts of elements used for vectorization. On GPU 1..2 should be fine, on CPU 4..16" )
+set(ISAAC_VECTOR_ELEM "1" CACHE STRING "The amounts of elements used for vectorization. On GPU 1 should be fine, on CPU 4..32, depending on the vectorization capabilities" )
```

Comment:
Exactly. I thought 2 is a good value for gpus, because of the possibility for out of order execution of code and stuff. But this just doesn't work and is much slower than with 1.

For my cpu btw. 16 is the value, where I get best results (incredible 8% speedup!).